src/commands/PublishCommand.js

```diff
@@ -24,21 +24,37 @@ export const command = "publish";
 
 export const describe = "Publish packages in the current project.";
 
+const cdVersionOptions = [
+  "major",
+  "minor",
+  "patch",
+  "premajor",
+  "preminor",
+  "prepatch",
+  "prerelease",
+];
+
+const cdVersionOptionString =
+  `'${cdVersionOptions.slice(0, -1).join("', '")}', or '${cdVersionOptions[cdVersionOptions.length - 1]}'.`;
+
 export const builder = {
   "canary": {
     group: "Command Options:",
     defaultDescription: "alpha",
     describe: "Publish packages after every successful merge using the sha as part of the tag.",
-    alias: "c"
+    alias: "c",
+    type: "string",
   },
   "cd-version": {
     group: "Command Options:",
-    describe: "Skip the version selection prompt and increment semver 'major', 'minor', or 'patch'.",
+    describe: `Skip the version selection prompt and increment semver: ${cdVersionOptionString}`,
     type: "string",
     requiresArg: true,
     coerce: (choice) => {
-      if (!["major", "minor", "patch"].some((inc) => choice === inc)) {
-        throw new Error(`--cd-version must be one of 'major', 'minor', or 'patch', got '${choice}'`);
+      if (!cdVersionOptions.includes(choice)) {
```

Comment:
We still support node v4, so we can't use `Array#includes()`. Can probably just restore the previous `.some()` iterator, or maybe just a plain old `if (cdVersionOptions.indexOf(choice) === -1) {` ? Not as fancy, but should get the job done.